### PR TITLE
Simplified Schemes and routeSelected Endpoint Fix

### PR DESCRIPTION
**Pull request by mattbarker016**
Exactly what the title says :). I made sure all the new schemes could build, and they do :) :) :)

### CODE REVIEW

File: TCAT.xcodeproj/xcshareddata/xcschemes/TCAT Debug.xcscheme [hunk 1]
```diff
@@ -82,10 +82,10 @@
       </BuildableProductRunnable>
    </ProfileAction>
    <AnalyzeAction
-      buildConfiguration = "Staging Debug">
+      buildConfiguration = "Debug">
    </AnalyzeAction>
    <ArchiveAction
-      buildConfiguration = "Staging Release"
+      buildConfiguration = "Debug"
```
Comment (Omarrasheed) [inline TCAT.xcodeproj/xcshareddata/xcschemes/TCAT Debug.xcscheme:88]: Just making sure but is this supposed to be debug or release?

File: TCAT.xcodeproj/xcshareddata/xcschemes/Today Extension Debug.xcscheme [hunk 2]
```diff
@@ -57,7 +57,7 @@
       </AdditionalOptions>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Staging Release"
+      buildConfiguration = "Debug"
```
Comment (Omarrasheed) [inline TCAT.xcodeproj/xcshareddata/xcschemes/Today Extension Debug.xcscheme:60]: I'm not 100% sure on our scheme related stuff but same comment here

**Review by Omarrasheed — APPROVED** [review]
lgtm. I'm kinda confused on how schemes work in general but I'll read up a lil more

**Review by kevinchan159 — APPROVED** [review]
looks great!

**Pull request merged**